README.md

```diff
@@ -138,6 +138,7 @@ Renders a test component that will call the provided `callback`, including any h
 
 - `result` (`object`)
   - `current` (`any`) - the return value of the `callback` function
+  - `error` (`Error`) - the error that was thrown if the `callback` function threw an error during rendering
```

Comment:
Technically you can throw anything (not just an `Error` instance), so I'm not sure if this should be `any` as well?

Reply:
I am not knowledgeable enough with typescript, but I think even if someone would throw, it would be an instance of `Error` as they would extend `Error` to create a custom error in case they wanted to. So, I think this should be fine unless someone with knowledge of typescript differs in opinion.